Testscripts/Linux/TVM-TestSecureBootCompatibility.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 0
+}
+
+UtilsInit
+homeDir=$(dirname "$0")
+
+if [ ! -e ${homeDir}/sbinfo ]; then
+    LogErr "sbinfo tool is not on the system"
+    SetTestStateAborted
+    exit 0
+fi
+
+chmod +x ${homeDir}/sbinfo
+
+SBEnforcementStage=$(sudo ${homeDir}/sbinfo | grep SBEnforcementStage | sed -e s/'  "SBEnforcementStage": '//)
+LogMsg "$SBEnforcementStage"
+
+if [[ "$SBEnforcementStage" == *"Secure Boot is not enforced"* ]]; then
```

Comment:
I don't understand two if-statement at 24 and 31. Are you trying to do "OR" operator?

Reply:
done